src/dmd/e2ir.d

```diff
@@ -175,6 +175,11 @@ private elem *callfunc(const ref Loc loc,
     elem *eside = null;
     elem *eresult = ehidden;
 
+    if (fd && fd.flags & FUNCFLAG.compileTimeOnly)
+    {
+        fd.error("may only be used for CTFE");
```

Comment:
Should we mention that it is annotated ? E.g.:
```suggestion
        fd.error("is annotated with `pragma(ctfe)` and may not be called at runtime");
```

Or something similar, so the user can look up `pragma(ctfe)` if they need to (e.g. they use library code).

Reply:
Thanks, changed that.